__tests__/integration.ts

```diff
@@ -0,0 +1,50 @@
+import * as MongoClient from "mongodb";
+
+import RWMutex from "../lib/RWMutex";
+
+const MONGO_URL = "mongodb://localhost:27017/test";
+const lockID = "lockID";
+const clientID = "1";
+
+describe("RWMutex", () => {
+  // Connect to the database
+  let collection = null;
+  beforeAll(async () => {
+    const db = await MongoClient.connect(MONGO_URL);
+    collection = db.collection("lock_test");
+    return await collection.createIndex("lockID", { unique: true });
+  });
+
+  // Reset the collection before each test
```

Comment:
Nope - Was going to add test coverage in a separate PR because we'd probably want thorough coverage